db/migrate/20170913165446_add_column_interested_intext_into_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddColumnInterestedIntextIntoUsers < ActiveRecord::Migration[5.1]
+  def up
+    add_column :users, :interested_in_other, :string
```

Comment:
I know the other column is called `interested_in`, so `interested_in_other` kinda makes sense to keep this aligned. However, I'm not really sold on that name 🙈 
Since we're dealing with user objects here, I'd rather name it `interests` and `other_interests`, since in both cases I assume to retrieve an array of interests.

Aside that, if we want to have this as a list of interests rather then prose text, we should probably align it with the other interests field and serialize it to an array 🤔  Or why did you decide against this?